_posts/2019-12-25-technical-debt.md

```diff
@@ -0,0 +1,70 @@
+
+> Debt certainly isn't always a bad thing. A mortgage can help you afford a home. Student loans can be a necessity in getting a good job. Both are investments worth making, and both come with fairly low interest rates. - Jean Chatzky
+
+Not all debt in life is necessarily bad and the same can be said of technical debt. Debt in service of an education has historically proven to be a sensible decision, as it opens up opportunities for returns that would not have been possible without it. A company that has never been profitable is  probably better off concentrating on becoming profitable then obsessing over their software architecture. Taking upon some debt in service of this goal is worthwhile. The caveat is this must be a conscious decision. It is unwise to take out a loan out if you will be unable to get a return-on-investment. Yes, there is always some level of uncertainty in a new venture and value is often found in the unknown, but taking no steps towards due-diligence is waving a white flag to negligence.
+
+If technical debt can't be measured in it's own units, must be informed by other metrics, and can be both good or bad, is it even worth talking about? It probably isn't helpful if you are fixated with tanglible outputs. Technical debt could function as a term to categorise team performance when restricted to a particular system, but why not just use "Team Performance"?. The term "technical debt" is useful as a barometer of team morale. Common usage is that of a perjorative, and so it signals how a team feels about the work it is producing. If discussions about technical debt are common in your team, it may be a good time to have a conversation about how the team feels about the trade-offs it is making. Are they the right trade-offs? Does the team feel valued? Does the team feel they are empowered and trusted to make the best decisions they can? Many studies have indicated that teams do their best when they have a sense of purpose and are empowered with the appropriate autonomy. If endless discussions about technical debt are becoming the main event at your daily standup and retrospectives, it may be a sign that there underlying issues regarding ownership that need to be addressed.
```

Comment:
> in it's own units

in its own units
---
first sentence reads weird with commas. not sure if it's any better but felt better to me like 

> If technical debt can't be measured in it's own units; must be informed by other metrics; can be both good or bad: is it even worth talking about? 

---
Is this paragraph back to front? Maybe you should start with a clickbaity sentence and then explain yourself, e.g.

> Teams hobbled by technical debt typically lack (a) Autonomy , (b) Leadership, or (c) Both

Reply:
I'll probably rearrange it somewhat. The point of the paragraph was that technical debt is fairly useless term on it's own; but it's presence (as in, your team talking about it all the time) is a good indication that a team is not satisfied with their work - and the team is probably in an a highly/higher emotional state over it.